pkg/cmd/reset.go

```diff
@@ -100,5 +105,43 @@ func runReset(opts *resetOpts) error {
 
 	s.Logger.Warnln("this command will require an explicit confirmation starting with the next minor release (v1.3)")
 
+	fmt.Println("The following nodes will be resetted: ")
+	for _, node := range s.Cluster.ControlPlane.Hosts {
+		fmt.Printf("\t+ reset control plane node %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+	for _, node := range s.Cluster.StaticWorkers.Hosts {
+		fmt.Printf("\t+ reset static worker nodes %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+
+	if !s.Cluster.MachineController.Deploy {
+		s.Logger.Info("Skipping deleting workers because machine-controller is disabled in configuration.")
+	}
```

Comment:
We already print that message, so I don't think this is needed here: https://github.com/kubermatic/kubeone/blob/af4f8c57d1b5d6fabd01c9694422d2347349274f/pkg/templates/machinecontroller/helper.go#L91-L95

Reply:
solved in: https://github.com/kubermatic/kubeone/pull/1251/commits/35dfd6ef9da62a0bac09e58db6b7c7aab36e0544

please have a look